libs/remix-solidity/src/compiler/compiler.ts

```diff
@@ -159,7 +161,7 @@ export class Compiler {
           source: source
         }
       }
-      this.event.trigger('compilationFinished', [true, data, source])
+      this.event.trigger('compilationFinished', [true, data, source, input, version])
```

Comment:
see line 151,  we should add `intput` and `version ` there too.

Reply:
done